source/index.html.erb

```diff
@@ -11,12 +11,13 @@ created_by:
   email: james@lazyatom.com
   name: James Adam
 layout: base
+description: The London Ruby User Group (LRUG) meets on the 2nd Monday of the month for talking and socialising (about Ruby).
 ---
 
 <%
   meetings = meeting_pages.take(10)
   featured_meeting = meetings.shift
 %>
-<%= partial 'meetings/meeting', locals: { meeting: featured_meeting, first: true } %>
+<%= partial 'meetings/meeting', locals: { meeting: featured_meeting, first: true, skip_auto_ids: true} %>
```

Comment:
Because Kramdown generates IDs for headers (which is awesome), it would create duplicate ids on the homepage. I added a way to pragmatically skip the generation of the header IDs based on context.